src/Lykke.Service.SwiftWithdrawal.Core/Domain/Cashout/ICashoutAttemptRepository.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Lykke.Service.SwiftWithdrawal.Contracts;
+
+namespace Lykke.Service.SwiftWithdrawal.Core.Domain.Cashout
+{
+    public interface ICashoutAttemptRepository
```

Comment:
A lot of not used methods, should be removed. This applies to other repositories also.

Reply:
removed